### ISSUE TITLE
Installation instructions

### ISSUE THREAD

**Issue by peteruithoven** [pop-os/hidpi-daemon#7]
Please include installation instructions for distributions other than Pop!_OS. 

I've tried:
```
git clone https://github.com/pop-os/hidpi-daemon.git
cd hidpi-daemon
./setup.py build
sudo ./setup.py install
```

From the logs I understand that it's added to `/usr/local/lib/python3.5/dist-packages/`, but I was also expecting a service to be added to SystemD (`/etc/systemd/system`), which didn't seem to happen. 

Should this daemon be installed from a ppa?

(I'm on elementaryOS Loki)

**Comment by brs17** [comment]
@djordan2 I definitely agree it would be a good idea to get some instructions posted for how one can use/setup/remove the hidpi-daemon especially since it is in it's own repository now.

**Comment by djordan2** [comment]
Good idea for install instructions.  Currently we've really only tooled it for installation via the debian packaging, but it would be nice to make setup.py work correctly.  There are some files that need to be in /usr/lib/hidpidaemon, /etc/xdg/autostart (because systemd user session services don't work right in gnome-session), and a glib schema.

The notification system in elementary OS functions differently from Gnome shell in a way that doesn't work well with the hidpi daemon's needs, and no other GUI has been written, so you'd either need to control the daemon using gsettings or write a frontend for elementary.  I know the elementary devs have been interested in this, but haven't had the time to write one.
